### PR TITLE
Update the slide caption for bkentel-mcafee

### LINKED ISSUE #7: bkentel-mcafee's checklist
- [x] Create a branch
- [x] Make some changes
- [x] Make a commit
- [x] Push our changes
- [x] View a PR
- [ ] Merge

@khiepleh-mfe let's get'er done

### PR DESCRIPTION
**Pull request by bkentel-mcafee**
This partially fixes #7

### CODE REVIEW

File: _posts/0000-01-15-bkentel-mcafee.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "🎉 Welcome to our slide deck! 🎉"
 ---
 
-Caption here
+Check out this amazing caption 😍😍😍
```
Comment (khiepleh-mfe) [inline _posts/0000-01-15-bkentel-mcafee.md]: I disagree that this caption is amazing. You only have _three_ emojis, and our weekly quota is _much_ higher than that.

😤😤😤😤😤

Comment (bkentel-mcafee) [reply _posts/0000-01-15-bkentel-mcafee.md]:
```suggestion
Check out this amazing caption 😍😍😍😍😍
```

Hows's that? Forgot about the quota 😭

Comment (bkentel-mcafee) [reply _posts/0000-01-15-bkentel-mcafee.md]: @apokluda what do you think?

Comment (apokluda) [reply _posts/0000-01-15-bkentel-mcafee.md]: That's better, but you really could use more variety. Using the same emoji repeatedly is not very creative.

Comment (bkentel-mcafee) [reply _posts/0000-01-15-bkentel-mcafee.md]: @apokluda I see your point -- let me revisit the change.

Comment (bkentel-mcafee) [reply _posts/0000-01-15-bkentel-mcafee.md]: @khiepleh-mfe  @apokluda 
How do the latest changes look?

Comment (ndavies-mcafee) [reply _posts/0000-01-15-bkentel-mcafee.md]: Now, you know it's up to you whether or not you want to just do the bare minimum. Or... well, like Brian, for example, has thirty seven emojis, okay. And a terrific smile.

**Pull request merged**